src/scripts/main.js

```diff
@@ -1 +1,72 @@
 'use strict';
+
+const body = document.querySelector('body');
+const div = document.createElement('div');
+let leftClickHappened = false;
+let rightClickHappened = false;
+
+div.setAttribute('data-qa', 'notification');
+body.appendChild(div);
```

Comment:
The created 'div' element is appended to the body but it doesn't seem to have any content or specific purpose. Make sure that this element is necessary or consider removing it if it's not being used.